docs/cmdline-opts/proxy-ciphers.md

```diff
@@ -2,25 +2,26 @@
 c: Copyright (C) Daniel Stenberg, <daniel@haxx.se>, et al.
 SPDX-License-Identifier: curl
 Long: proxy-ciphers
-Arg: <list>
-Help: SSL ciphers to use for proxy
+Arg: <list of ciphers>
```

Comment:
Since the option name includes "ciphers" I figured an optimization to keep this short is to not include "ciphers" again in the argument. The help output needs to be as short as possible.